### PR TITLE
Added Resume field in the work section

### PR DESCRIPTION
**Pull request by nidhi-bhanushali**
Added Resume field between blog and funfact in the following issue.
https://github.com/rahuldkjain/github-profile-readme-generator/issues/125#issue-708667271

### CODE REVIEW

**Review by rahuldkjain — CHANGES_REQUESTED** [review]
Please integrate the resume field in `src/components/markdown.js` and `src/components/markdownPreview.js`

**Comment by nidhi-bhanushali** [comment]
On it!

**Comment by nidhi-bhanushali** [comment]
Hey , @rahuldkjain , I've made the changes , so are they approved?
You haven't closed the issue yet , so

File: package.json [hunk 1]
```diff
@@ -12,12 +12,13 @@
...
     "gatsby-plugin-offline": "^3.2.13",
     "gatsby-plugin-react-helmet": "^3.3.6",
-    "gatsby-plugin-sharp": "^2.6.14",
+    "gatsby-plugin-sharp": "^2.6.38",
     "gatsby-remark-prismjs": "^3.5.10",
     "gatsby-source-filesystem": "^2.3.23",
     "gatsby-transformer-remark": "^2.8.27",
     "gatsby-transformer-sharp": "^2.5.7",
     "gsap": "^3.4.0",
+    "netlify-cli": "^2.64.1",
```
Comment (rahuldkjain) [inline package.json:21]: please remove `netlify-cli` from `package.json`.

File: package.json [hunk 2]
```diff
@@ -12,12 +12,13 @@
     "gatsby-plugin-manifest": "^2.4.14",
     "gatsby-plugin-offline": "^3.2.13",
     "gatsby-plugin-react-helmet": "^3.3.6",
-    "gatsby-plugin-sharp": "^2.6.14",
+    "gatsby-plugin-sharp": "^2.6.38",
```
Comment (rahuldkjain) [inline package.json:15]: Please don't change the version.

File: .netlify/state.json [hunk 3]
```diff
@@ -0,0 +1,3 @@
+{
```
Comment (rahuldkjain) [inline .netlify/state.json:1]: remove this file

File: package.json (hunk 2 above)
Comment (rahuldkjain) [inline package.json:15]: Please don't change the version.

File: package.json (hunk 1 above)
Comment (rahuldkjain) [inline package.json:21]: remove `netlify-cli` from `package.json`

**Review by rahuldkjain — CHANGES_REQUESTED** [review]
Please fix the respective comments.

**Comment by rahuldkjain** [comment]
@nidhi-bhanushali delete `package-lock.json` file too from this PR.

**Pull request merged**